doc.html

```diff
@@ -16,1281 +16,982 @@
+                        </p>
+                        <p class="info_text" style="text-align:justify">
+                            It then pipes the output of the benchmark into pbench-tool-trigger:
+                        </p>
+                        <p class="c_snip" style="text-align:initial">
+                            $benchmark_bin --machine-readable --directory=$dir --timelimit=$runtime
+                            <br>&nbsp; &nbsp; &nbsp; &nbsp;--warmup=$warmup --loadfile $loadfile
+                            $client |
+                            <br>&nbsp; &nbsp; &nbsp; &nbsp; tee $benchmark_results_dir/result.txt |
+                            <br>&nbsp; &nbsp; &nbsp; &nbsp; pbench-tool-trigger "$iteration"
+                            "$benchmark_results_dir" no
+                        </p>
+                        <p class="info_text" style="text-align:justify">
+                            pbench-tool-trigger will then start the warmup group when it encounters
+                            the string "warmup" in the benchmark's output and stop
+                            it when it encounters "execute". It will also start the measurement
+                            group when it encounters "execute" and stop it when it
+                            encounters "cleanup" - and so on.
+                        </p>
+                        <p class="info_text" style="text-align:justify">
+                            Obviously, the start/stop conditions will have to be chosen with some
+                            care to ensure correct actions.
+                        </p>
                     </div>
                 </div>
-            </div> -->
-            <div class="section">
-                <span class="footer">
+            </section>
+            <div class="footer" style="position: sticky;margin-top: 0px;">
                 <p class="darkFoot">2020 Pbench | Privacy Statement</p>
                 <p class="lightFoot">Code licensed under a thing, site under a thing</p>
-                </span>
-            </div>  	
+            </div>
         </main>
-</div>
-
+    </div>
 </body>
 <script src="./js/jquery.min.js"></script>
-<script src="./js/tab.js"></script>
-<script src="./js/accordion.js"></script>
+<!-- <script src="./js/tab.js"></script>
+<script src="./js/accordion.js"></script> -->
 <script>
-var heightContent = $("#agentUserGuideToc").height();
-document.getElementById("agentUserGuideContent").style.height = heightContent+'px';
-function toggleSearchIcon(){
-    $(".searchBtn").hide();
-    if($(".searchInput").val()===''){
-        $(".searchBtn").show();
+    var heightContent = $("#agentUserGuideToc").height();
+    document.getElementById("agentUserGuideContent").style.height = heightContent + 'px';
+    function toggleSearchIcon() {
+        $(".searchBtn").hide();
+        if ($(".searchInput").val() === '') {
+            $(".searchBtn").show();
+        }
     }
-}
 </script>
-</html>
+
+</html>
```

Comment:
You lost the newline at the end of the file. If only to avoid the weird GitHub icon, it'd be nice to have it back.